packages/amplication-client/src/Resource/create-resource/CreateServiceWizard.scss

```diff
@@ -41,7 +41,7 @@ $max-width: 700px;
 
     .circle-badge i {
       color: var(--black5);
-      font-size: 14px;
+      font-size: 20px;
```

Comment:
Is that relevant?

Reply:
@yuval-hazaz  I fixes this in another pr , i will remove it